notebooks/02_TimeIntegration/02_01_EulerMethod.md

```diff
@@ -137,10 +137,10 @@ nt = int((tf-ti)/dt)
 # over np.zeros.
 N = np.empty(nt+1)
 
-# We pass the initial condition to the N array,
+# We pass the initial condition to the N array...
```

Comment:
As I said, there are also minor visual changes, and while reading the notebook, I felt like having something else than a comma at the end of the commented line, like a common dot, or here and ellipsis since the sentence is closed in the next comment. It's not a principal change, so if you prefer the old version, feel free to keep it !

Reply:
Thanks, I have removed the ellipses as we don't use it often. But still changed a bit the puncutation as the original one was not that good.